src/VisualStudio/Core/Next/Remote/RemoteHostClientFactory.cs

```diff
@@ -15,20 +14,20 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
     [ExportWorkspaceService(typeof(IRemoteHostClientFactory)), Shared]
     internal class RemoteHostClientFactory : IRemoteHostClientFactory
     {
-        public Task<RemoteHostClient> CreateAsync(Workspace workspace, CancellationToken cancellationToken)
+        public async Task<RemoteHostClient> CreateAsync(Workspace workspace, CancellationToken cancellationToken)
         {
             try
             {
                 // this is the point where we can create different kind of remote host client in future (cloud or etc)
-                return ServiceHubRemoteHostClient.CreateAsync(workspace, cancellationToken);
+                return await ServiceHubRemoteHostClient.CreateAsync(workspace, cancellationToken).ConfigureAwait(false);
```

Comment:
try/catch doesn't work if you just return the Task.  it will only catch exceptions up to the first 'await' in the async method.  Any further exceptions will be missed.  To actually get properly exception handling, you have to await the call so that the entirety of the async work is executed.